lib/main.coffee

```diff
@@ -9,6 +9,11 @@ module.exports =
       type: 'boolean'
       description: 'and a .sass-lint.yml file is not specified in the .sass-lint.yml Path option'
       default: false
+    resolvePathsRelativeToConfig:
+      title: 'Resolve paths in configuration relative to config file'
+      type: 'boolean'
+      description: '(and not relative to project root)'
```

Comment:
Would you mind changing this to `Instead of the default where paths are resolved relative to the project root`